tests/Controller/EnMarche/ReferentControllerTest.php

```diff
@@ -18,6 +18,7 @@
 /**
  * @group functional
  * @group referent
+ * @group wip
```

Comment:
:D 
try phpstorm to run one isolated test method :)

Reply:
Rohhh my bad, sorry.

Reply:
Another way, you can run it with :  

`docker-compose run --rm app vendor/bin/phpunit --filter ReferentControllerTest`

with the class / method name for the filter